src/http_rest_api_handler.cpp

```diff
@@ -497,6 +498,33 @@ Status HttpRestApiHandler::prepareGrpcRequest(const std::string modelName, const
     return StatusCode::OK;
 }
 
+static std::set<std::string> getRequestedBinaryOutputs(::KFSRequest& grpc_request, ::KFSResponse& grpc_response) {
```

Comment:
The function name states that you are extracting requested outputs which is misleading. My suggestion is to change it to `queryOutputNamesToRepresentAsBinary`